src/clients/wol.rb

```diff
@@ -140,11 +142,11 @@ def main
           @overview = WOL.Overview
           UI.ChangeWidget(Id(:table), :Items, @overview)
         elsif @ret == :wake
-          @mac = Convert.to_string(UI.QueryWidget(Id(:table), :CurrentItem))
-          if @mac != nil && @mac != ""
-            @cmd = Builtins.sformat("/usr/bin/wol %1", @mac)
-            Popup.ShowFeedback(_("Waking remote host"), @mac)
-            SCR.Execute(path(".target.bash"), @cmd)
+          mac = Convert.to_string(UI.QueryWidget(Id(:table), :CurrentItem))
+          if mac != nil && mac != ""
```

Comment:
Just a question: why not use `to_s` directly? I mean, since you are also changing the previous line, why not to remove `Convert.to_string` and use `to_s` instead. Then, you can simplify the `if` statement.

Reply:
this is true. I just want to do minimal changes that I consider safe enough.